RecoMET/METProducers/src/HcalNoiseInfoProducer.cc

```diff
@@ -380,7 +386,8 @@ HcalNoiseInfoProducer::filldigis(edm::Event& iEvent, const edm::EventSetup& iSet
     edm::RefVector<HBHERecHitCollection> &rechits=hpd.rechits_;
     for(edm::RefVector<HBHERecHitCollection>::const_iterator rit=rechits.begin();
 	rit!=rechits.end(); ++rit, ++counter) {
-      if((*rit)->id() == digi.id()) {
+      const HcalDetId & detid = (*rit)->idFront();
+      if(DetId(detid) == digi.id()) {
```

Comment:
I believe this will be revised later - we would like to have a working version so that the workflow PR can be submitted